std/format.d

```diff
@@ -2350,8 +2482,13 @@ private void formatChar(Writer)(Writer w, in dchar c, in char quote)
         formattedWrite(w, "\\U%08X", cast(uint)c);
 }
 
-// undocumented
```

Comment:
I believe this means the function is deliberately undocumented. So please leave this line there. (Though it's probably harmless, even good, to document Params: regardless.)